lib/conversion.ts

```diff
@@ -35,3 +35,59 @@ export function convertedObjectIDInSearch(
 
   this.sendEvent("conversion", params as InsightsEvent);
 }
+
+export interface InsightsSearchConversionObjectIDsEvent {
```

Comment:
Why do we have the Search inside the name? This event is not tied to a search right?

Reply:
fixed in next-next commit!